lib/services/authentication.go

```diff
@@ -111,37 +119,62 @@ type AuthPreferenceV2 struct {
 	Spec AuthPreferenceSpecV2 `json:"spec"`
 }
 
-// SetExpiry sets expiry time for the object
+// GetVersion returns resource version.
+func (c *AuthPreferenceV2) GetVersion() string {
+	return c.Version
+}
+
+// GetName returns the name of the resource.
+func (c *AuthPreferenceV2) GetName() string {
+	return c.Metadata.Name
+}
+
+// SetName sets the name of the resource.
+func (c *AuthPreferenceV2) SetName(e string) {
+	c.Metadata.Name = e
+}
+
+// SetExpiry sets expiry time for the object.
 func (c *AuthPreferenceV2) SetExpiry(expires time.Time) {
 	c.Metadata.SetExpiry(expires)
 }
 
-// Expirey returns object expiry setting
+// Expiry returns object expiry setting.
 func (c *AuthPreferenceV2) Expiry() time.Time {
 	return c.Metadata.Expiry()
 }
 
-// GetResourceID returns resource ID
+// SetTTL sets Expires header using realtime clock.
```

Comment:
nit: I know this has been copy/pasted but I'd remove the `realtime` mention - it might as well be a fake clock to enable testing.

Reply:
Right, almost all of the `SetTTL` comments make this reference to "realtime clock" except for one that mentions "current clock" which would be prone to your criticism as well.

If
```
// SetTTL sets Expires header using the provided clock.
```
sounds better to you, I can change it en block as part of a small collection of similar code style/quality improvements I already have and create another PR.